### DISCUSSION TITLE
평생 무료로 커스텀 이메일 사용하기 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#24] [Blog Comments]
# 평생 무료로 커스텀 이메일 사용하기 - peterkimzz

안녕하세요. 또 다시 찾아온 평생 무료 시리즈입니다.
저는 틈만나면 1인 사이드 프로젝트를 진행하기 때문에, 어떻게든 공짜로 서버를 돌리기 위해 온갖 노력을 하고 있습니다. 그래서 무료로 이용하는 방법에 관한 글을 몇 개 올렸는데 GA를 살펴보니 다른 주제보다 조회수가

https://www.peterkimzz.com/custom-email-service-for-free-forever/

**Comment by linusdamyo** [comment]
이메일 서비스가 필요했는데, 덕분에 너무 잘 설정했어요!
정말 감사드려요!

**Reply by peterkimzz** [reply]
도움이 되셨다니 정말 뿌듯하네요! 감사합니다.
